osquery/filesystem/fileops.h

```diff
@@ -202,9 +202,9 @@ class PlatformFile {
    * @brief Determines how immutable the file is to external modifications.
    * @note Currently, this is only implemented on Windows. The Windows version
    *       of this function ensures that writes are explicitly denied for the
-   *       file and the file's parent directory.
+   *       file AND the file's parent directory.
    */
-  Status isNonWritable() const;
+  Status isImmutable() const;
```

Comment:
I'd have to check how it's used. If we're only calling it for the exec/fork flow to make sure no one could have swapped out the binary from under our nose-- that works :)